interp/src/interpreter/interpret_group.rs

```diff
@@ -196,300 +141,185 @@ fn get_done_port(group: &ir::Group) -> RRC<ir::Port> {
 }
 
 // XXX(Alex): Maybe rename to `eval_is_done`?
-fn grp_is_done(done: OutputValueRef) -> bool {
+fn signal_is_high(done: OutputValueRef) -> bool {
     match done {
         OutputValueRef::ImmediateValue(v) => v.as_u64() == 1,
         OutputValueRef::LockedValue(_) => false,
         OutputValueRef::PulseValue(v) => v.get_val().as_u64() == 1,
     }
 }
 
-pub fn interp_cont(
-    continuous_assignments: &[ir::Assignment],
+pub fn interp_assignments<'a, I: Iterator<Item = &'a ir::Assignment>>(
     env: Environment,
-    comp: &ir::Component,
+    done_signal: &ir::Port,
+    assigns: I,
 ) -> FutilResult<Environment> {
-    // TODO: Set go to high
-    // go find go port for given assignment
-    let dependency_map = continuous_assignments.iter().into();
+    let assigns = assigns.collect::<Vec<_>>();
```

Comment:
(No need for resolution) Can also use `itertools::collect_vec()`: https://docs.rs/itertools/0.10.1/itertools/trait.Itertools.html#method.collect_vec

Reply:
Ah! Be still my quaking heart